DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/SetDelegatePassword/Index.cshtml

```diff
@@ -45,7 +46,8 @@
       var routeData = new Dictionary<string, string> { { "delegateId", Model.DelegateId.ToString() } };
       <vc:cancel-link asp-controller="ViewDelegate" asp-action="Index" asp-all-route-data="@routeData" />
     } else {
-      <vc:cancel-link asp-controller="AllDelegates" asp-action="Index" />
+      var routeData = new Dictionary<string, string> { { "page", Model.ReturnPage.ToString() } };
+      <vc:cancel-link asp-controller="AllDelegates" asp-action="Index" asp-all-route-data="@routeData" />

```

Comment:
I was giving a few links a quick test to see if they worked as I expected, and this one didn't seem to function for me.

Reply:
I forgot to set returnPage in the constructor 🤦‍♂️
Can't find anywhere else where I've made a similar mistake.